apps/core/scenes/passengerForm/BaggageBundles/BaggageTypeDetail.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, Icon } from '@kiwicom/universal-components';
+import type { IconNameType } from '@kiwicom/universal-components';
+import { Text } from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { bags, type Bag } from './BagTypes';
+
+type Props = {|
+  +bagType: Bag,
+  +dimensions?: string,
+  +quantity: number,
+  +chosen: boolean,
+  +weight: string,
+|};
+
+function getIcon(bagType, quantity: number): IconNameType {
```

Comment:
I think `quantity` number type shouldn't be necessary here.

Reply:
Or should I just add type for the bagType (I think I forgot to add it)?
Because I think its nice to have arguments' types in the functions

Reply:
I think it shouldn't be there if it's not necessary by Flow. Normally Flow should know the types according to function use.

Reply:
Alright, will remove it